kotlin/src/main/java/app/rive/runtime/kotlin/RiveArtboardRenderer.kt

```diff
@@ -87,6 +85,19 @@ open class RiveArtboardRenderer(
         }
     }
 
+    fun _advanceStateMachineInstance(
+        stateMachineInstance: StateMachineInstance,
```

Comment:
Kotlin doesn't really like the _ at the start, can't we just make it private?

Reply:
Yeah I think I saw a few warnings about this in Android Studio...just nitpicks really